src/components/DocumentationTopic.vue

```diff
@@ -80,10 +81,14 @@
           :class="{ 'no-primary-content': !hasPrimaryContent && enhanceBackground }"
         >
           <div
-            v-if="hasPrimaryContent"
+            v-if="hasPrimaryContent || hasOtherDeclarations"
```

Comment:
Not very important but should having these "other declarations" just be considered in the computation of `hasPrimaryContent` itself?

Reply:
Kept it separate because I still want the dividing line to appear when there are other declarations, but no primary content (for showing the button on the line). So don't want to apply the `no-primary-content` class above.